20241014-wordcloud/src/app.html

```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8" />
```

Comment:
I think, [technically](https://stackoverflow.com/a/19510239), since you're using the HTML 5 doctype, you should drop the trailing `/` on your `meta` and `link` tags.

Reply:
I think is a valid comment. You can prefix it with NIT to indicate is minor